mlir/tools/mlir-query/mlir-query.cpp

```diff
@@ -21,9 +21,9 @@
 using namespace mlir;
 
 // This is needed because these matchers are defined as overloaded functions.
-using HasOpAttrName = detail::AttrOpMatcher(StringRef);
-using HasOpName = detail::NameOpMatcher(StringRef);
-using IsConstantOp = detail::constant_op_matcher();
+using HasOpAttrName = mlir::detail::AttrOpMatcher(StringRef);
+using HasOpName = mlir::detail::NameOpMatcher(StringRef);
+using IsConstantOp = mlir::detail::constant_op_matcher();
 
```

Comment:
Unrelated changes?

Reply:
I'm not sure... For some reason these are reported as ambiguous if I don't prefix them with `mlir::` now, not worked out why yet.